lib/kapost_deploy/plugins/notify_honeybadger_after_promote.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require "kapost_deploy/heroku/app_releases"
+
+module KapostDeploy
+  module Plugins
+    # After-promotion plugin to notify honeybadger after a promotion is complete.
+    # Honeybadger.io can be configured to clear all honeybadgers upon this deploy notification.
+    class NotifyHoneyBadgerAfterPromote
```

Comment:
Agree that we might want to remove the `AfterPromote`.  I was looking at the [terminology they use in their docs](http://docs.honeybadger.io/lib/ruby.html#deployment-tracking), and maybe we do something that matches more closely like `HoneybadgerDeploymentTrackingPlugin` or `TrackDeploymentInHoneybadgerPlugin`.